playwright-e2e/tests/osteo/osteo-regression-test.spec.ts

```diff
@@ -0,0 +1,756 @@
+import { test, expect } from '@playwright/test';
+import { setAuth0UserEmailVerified } from 'utils/api-utils';
+import { APP } from 'data/constants';
+import { generateEmailAlias } from 'utils/faker-utils';
+import * as testutils from 'utils/test-utils';
+import ResearchConsentPage  from 'pages/osteo/consent-page';
+import HomePage from 'pages/osteo/home-page';
+import PrequalPage from 'pages/osteo/prequal-page';
+import ConsentAddendumPage from 'pages/osteo/consent-addendump-page';
+import ConsentAssentPage from 'pages/osteo/consent-assent-page';
+import * as auth from 'authentication/auth-angio';
+import ReleasePage from 'pages/osteo/release-page';
+import AboutYourOsteosarcoma from 'pages/osteo/about-your-osteosarcoma-page';
+import { logParticpantCreated } from 'utils/log-utils';
+import { lastIndexOf } from 'lodash';
+import { faker } from '@faker-js/faker';
+import { generateUserName } from 'utils/faker-utils';
+import { CancerSelector } from 'pages/cancer-selector';
+import { FamilyHistory } from 'pages/family-history';
+import { checkUserReceivedEmails } from 'utils/email-utils';
+
+const { OSTEO_USER_EMAIL, OSTEO_USER_PASSWORD, SITE_PASSWORD } = process.env;
+
+test('Static Content', async ({page}) => {
```

Comment:
We're grouping different study's tests by adding special tags like `@osteo` so we can run only test suites belongs to targeted study, not everything everytime.
For example,
`test('Static Content @osteo', async ({page}) => {`